lib/cron.js

```diff
@@ -582,10 +582,11 @@
 	CronJob.prototype.addCallback = addCallback;
 
 	CronJob.prototype.setTime = function(time) {
-		if (!(time instanceof CronTime))
+		if (typeof time !== 'object') // crontime is an object...
```

Comment:
I think it would make more sense to keep what was there and add `|| typeof time === 'string'`. What do you think?

Reply:
Looks good... but time should be an object returned by CronTime

`var time = new cron.CronTime('* * * * * *'); // return an object`

Reply:
Actually, I think this is pretty consistent with what's there now and if I ever get around to a bigger refactor, I can change it later.

I'll land this. Thanks.